dashboard/Deposits.tsx.preview

```diff
@@ -0,0 +1,14 @@
+<React.Fragment>
```

Comment:
How does this work? Is this the preview you use for testing?

Reply:
I have deleted this file with no effect, and cannot find any information about `tsx.preview` file.